src/Compilers/CSharp/Portable/Symbols/CollectionArgumentsSignatureOnlyMethodSymbol.cs

```diff
@@ -0,0 +1,152 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Diagnostics;
+using Microsoft.CodeAnalysis.PooledObjects;
+
+namespace Microsoft.CodeAnalysis.CSharp.Symbols
+{
+    /// <summary>
+    /// A method symbol that represents a signature for collection expression arguments.
+    /// The compiler is free to emit any equivalent method call or initialization.
+    /// </summary>
+    internal sealed class CollectionArgumentsSignatureOnlyMethodSymbol : MethodSymbol
```

Comment:
>CollectionArgumentsSignatureOnlyMethodSymbol [](http://example.com/codeflow?start=26&length=44)

Is there any way to observe these symbols outside the compiler?

Reply:
I don't think so currently, but we do need to determine what is returned from `SemanticModel` and `IOperation`.